pkgs/applications/misc/shoutrrr/default.nix

```
@@ -0,0 +1,22 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "shoutrrr";
+  version = "0.8.0";
```

follows my comment about `rec`
```suggestion
  inherit version;
```

---

done